CodeWars Day 3/Solution3.js

```diff
@@ -0,0 +1,20 @@
+function insertDash(num) {
+  let prev='', 
+   newstring='';
+ 
+   let insert = num.toString();
+   
+   for (let i=0; i<insert.length; i++){
+     parseInt(insert[i])%2 == 0 ? current='even' : current='odd';
```

Comment:
In some places you have spaces on both sides of your equals and in other places you don't. Make sure to stay consistent with which one you're are doing, most people generally put spaces on both sides because it spreads things out a little and makes it easier to read.